README.md

```diff
@@ -988,6 +989,52 @@ targets:
     releaseRepoName: 'unity'
 ```
 
+### Maven central (`maven`)
+
+PGP signs and publishes packages to Maven Central.
+
+**Environment**
+
+| Name                     | Description                      |
+| ------------------------ | -------------------------------- |
+| `OSSRH_USERNAME`         | Username of Sonatype repository. |
+| `OSSRH_PASSWORD`         | Password of Sonatype repository. |
+| `MAVEN_CENTRAL_USERNAME` | Username of Maven Central.       |
+| `MAVEN_CENTRAL_PASSWORD` | Password of Maven Central.       |
+
+**Configuration**
+
+| Option              | Description                                                           |
+| ------------------- | --------------------------------------------------------------------- |
+| `gradleCliPath`     | Path to the Gradle CLI. It must be executable by the calling process. |
+| `mavenCliPath`      | Path to the Maven CLI. It must be executable by the calling process.  |
+| `mavenSettingsPath` | Path to the Maven `settings.xml` file.                                |
+| `mavenRepoId`       | ID of the Maven server in the `settings.xml`.                         |
+| `mavenRepoUrl`      | URL of the Maven repository.                                          |
+| `android`           | Structure containing the data available below.                        |
+
+The `android` structure contains the following options:
+
+- `distDirRegex`: pattern of distribution directory names.
+- `fileReplaceeRegex` :pattern of substring of distribution module names to be replaced to get the Android distribution file.
+- `fileReplacerStr`: string to be replaced in the module names to get the Android distribution file.
+
+**Example**
+
+```yaml
+targets:
+  - name: maven
+    gradleCliPath: ./gradlew
+    mavenCliPath: scripts/mvnw.cmd
+    mavenSettingsPath: scripts/settings.xml
+    mavenRepoId: ossrh
+    mavenRepoUrl: https://oss.sonatype.org/service/local/staging/deploy/maven2/
+    android:
+      distDirRegex: /^sentry-android-.*$/
+      fileReplaceeRegex: /\d\.\d\.\d(-SNAPSHOT)?/
+      fileReplacerStr: release.aar
```

Comment:
just a heads up, https://github.com/getsentry/sentry-android-gradle-plugin publishes to maven central but it's not an Android project, it's a pure Java file (jars), so the `android` config. should be optional.

Reply:
Support for https://github.com/getsentry/sentry-android-gradle-plugin will come later.